src/app/shared/components/carriage/carriage.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="title-row">
+  <div class="number">Car {{ carriageNumber }}</div>
+  <div class="free-seats">{{ countOfFreeSeats }} free seats</div>
+</div>
+<div class="carriage">
+  <ng-container *ngFor="let seatGroup of [seats | splitSeats: leftSeats : rightSeats]">
+    <div class="seats seats-right">
+      <div *ngFor="let column of seatGroup[0]" class="column">
```

Comment:
It will be better if you use ngFor directive with trackBy function. Also, you can make it without implementing function in component.ts using @for (item of items; track item.name) {}, but do it the way you like more

Reply:
Thank you! I added it for:
```
 *ngFor="let seat of column; trackBy: trackBySeat"
```
 because seats have a unique identifier - the seat number